core/src/test/java/com/linecorp/armeria/server/websocket/WebSocketServiceHandshakeTest.java

```diff
@@ -192,7 +194,10 @@ void http2Handshake(SessionProtocol sessionProtocol) {
 
         final RequestHeadersBuilder headersBuilder =
                 RequestHeaders.builder(HttpMethod.CONNECT, "/chat")
-                              .add(HttpHeaderNames.PROTOCOL, HttpHeaderValues.WEBSOCKET.toString());
+                              .add(HttpHeaderNames.PROTOCOL, HttpHeaderValues.WEBSOCKET.toString())
+                              // It works even if the header contains multiple values
+                              .add(HttpHeaderNames.CONNECTION, HttpHeaderValues.KEEP_ALIVE.toString())
+                              .add(HttpHeaderNames.UPGRADE, "additional_value");
```

Comment:
Let's revert this change. I realized that HTTP/2 uses the different method so we probably don't need to check for HTTP/2.